psutil/TODO.aix

```diff
@@ -0,0 +1,30 @@
+AIX support is experimental and incomplete at this time.
+The following functions and methods are UNSUPPORTED on the AIX platform:
+
+    psutil.Process.memory_maps
+    psutil.Process.num_ctx_switches
+
+
+Process.io_counters read count is always 0
+
+
+TestSystemAPIs.test_pid_exists_2           there are pids in /proc that don't really exist
+TestProcess.test_name                      isolated python calls execve which changes process name
+TestProcess.test_num_fds                   opening a socket doesn't create fd in /proc/pid/fd (until data is sent??)
+TestProcess.test_open_files                /dev/null shows in open_files but it isn't a file
+TestProcess.test_pid_0                     pid 0 doesn't have a name on AIX
```

Comment:
Does `ps` list it? If not is there some other task manager on AIX that does? If not I would advice to return `"kernel"`.

Reply:
"ps" calls PID 0 "swapper". I can return this name specifically for the case of pid=0.

Reply:
Sounds good. Also make sure `pids()` return PID 0 and `pid_exists(0)` return True. If they don't by default then you can simply hard-code it. `ps` should be used as the leading reference and psutil should behave the same as ps.